modules/es-cluster-health-is-red.adoc

```diff
@@ -0,0 +1,153 @@
+
+.Procedure
+
+. Check the Elasticsearch cluster health and verify that the cluster `status` is red by running the following command:
++
+[source,terminal]
+----
+$ oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- health
+----
+
+. List the nodes that have joined the cluster by running the following command:
++
+[source,terminal]
+----
+$ oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_util --query=_cat/nodes?v
+----
+
+. List the Elasticsearch pods and compare them with the nodes in the command output from the previous step, by running the following command:
++
+[source,terminal]
+----
+$ oc -n openshift-logging get pods -l component=elasticsearch
+----
+
+. If some of the Elasticsearch nodes have not joined the cluster, perform the following steps.
+
+.. Confirm that Elasticsearch has an elected control plane node by running the following command and observing the output:
++
+[source,terminal]
+----
+$ oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_util --query=_cat/master?v
+----
+
+.. Review the pod logs of the elected control plane node for issues by running the following command and observing the output:
++
+[source,terminal]
+----
+$ oc logs <elasticsearch_master_pod_name> -c elasticsearch -n openshift-logging
+----
+
+.. Review the logs of nodes that have not joined the cluster for issues by running the following command and observing the output:
++
+[source,terminal]
+----
+$ oc logs <elasticsearch_node_name> -c elasticsearch -n openshift-logging
+----
+
+. If all the nodes have joined the cluster, check if the cluster is in the process of recovering by running the following command and observing the output:
++
+[source,terminal]
+----
+$ oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_util --query=_cat/recovery?active_only=true
+----
++
+If there is no command output, the recovery process might be delayed or stalled by pending tasks.
+
+. Check if there are pending tasks by running the following command and observing the output:
++
+[source,terminal]
+----
+$ oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- health |grep  number_of_pending_tasks
+----
+
+. If there are pending tasks, monitor their status. If their status changes and indicates that the cluster is recovering, continue waiting. The recovery time varies according to the size of the cluster and other factors. Otherwise, if the status of the pending tasks does not change, this indicates that the recovery has stalled.
+
+. If it seems like the recovery has stalled, check if the `cluster.routing.allocation.enable` value is set to `none`, by running the following command and observing the output:
++
+[source,terminal]
+----
+$ oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_util --query=_cluster/settings?pretty
```

Comment:
```suggestion
$ oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_util --query="_cluster/settings?pretty"
```

Reply:
Per https://redhat-internal.slack.com/archives/C02321AKYTT/p1696866637850469?thread_ts=1696524477.252959&cid=C02321AKYTT we will leave this as is